build.sh

```diff
@@ -55,22 +55,22 @@ fi
 
 CURRENT_DATE=$(date +%Y%m%d)
 
-case "$IB_SUITE" in
-  byzantium)
-    dist_version="10"
-    dist_reltag="~devel"
-  ;;
-  amber)
-    dist_version="9.0"
-  ;;
-  *)
-    echo "WARNING: Suite $SUITE is unknown" >&2
-    dist_version="0.0"
-  ;;
-esac
-VERSION_FULL="${dist_version}${dist_reltag}"
+# case "$IB_SUITE" in
```

Comment:
Why is this disabled

Reply:
want the eos version not the pureos version